src/TriggerService.Tests/IntegrationTests.cs

```diff
@@ -104,7 +104,7 @@ public async Task RunGlobTestWdlAsync()
         public async Task RunScaleTestWithMutect2Async()
         {
             const int countOfWorkflowsToRun = 100;
-            const string triggerFile = "https://raw.githubusercontent.com/microsoft/gatk4-somatic-snvs-indels-azure/main-azure/mutect2.trigger.json";
+            const string triggerFile = "https://raw.githubusercontent.com/microsoft/CromwellOnAzure/task/move-mutect-to-south-central/src/TriggerService.Tests/test-wdls/mutect2/mutect2.trigger.json";
```

Comment:
I assume that `/task/move-mutect-to-south-central/` needs to be `/main/` when this is committed to main, making this uri `https://raw.githubusercontent.com/microsoft/CromwellOnAzure/main/src/TriggerService.Tests/test-wdls/mutect2/mutect2.trigger.json`. Correct?